examples/contrib/modbus_mapper.py

```diff
@@ -85,11 +137,11 @@ def csv_mapping_parser(path, template):
     mapping_blocks = defaultdict(dict)
     with open(path, "r") as handle:  # pylint: disable=unspecified-encoding
         reader = csv.reader(handle)
-        reader.next()  # skip the csv header
+        next(reader)  # skip the csv header
```

Comment:
```suggestion
        reader.next()  # skip the csv header
```
We use this standard all over, so if you want if differently you need some good arguments.

Reply:
If I used that, python3 crashed.

Reply:
@janiversen I guess you meant `reader.__next__()` ?
In case of the csv reader the docs say `next(reader)` should be used instead of this way.

https://docs.python.org/3/library/csv.html?highlight=csv#csv.csvreader.__next__